packages/flutter/lib/src/widgets/mimic.dart

```diff
@@ -40,6 +44,12 @@ class MimicOverlayEntry {
   Curve _curve;
   Performance _performance;
 
+  /// Animate the entry to the location of the widget that has the given target key.
+  ///
+  /// The animation will take place over the given duration and will apply the
+  /// given curve.
+  ///
+  /// Currently we don't support calling this function more than once per overlay entry.
```

Comment:
I'd replace that last sentence with "This function can only be called once per overlay entry". Documentation should present an air of confidence. :-)